infra/cifuzz/example_main.yml

```diff
@@ -22,5 +18,5 @@ jobs:
       uses: actions/upload-artifact@v1
       if: failure()
       with:
-        name: fuzzer_testcase
-        path: ./out/testcase
+        name: bug_report
```

Comment:
How hard what it be to name this "crashes"?
It's possible we will have multiple anyway.

Reply:
We would have to send a new pull request to 4 different projects including zstd which we would have to get a FB employee to do.

Reply:
I think we decided offline that we will support both for now and eventually well deprecate the old behavior and force users to migrate to move over.

Reply:
Changed our minds offline to just switch now.